client/src/components/brushableHistogram/index.js

```diff
@@ -620,7 +621,10 @@ class HistogramBrush extends React.PureComponent {
     const summary = column.summarize();
     const range = [summary.min, summary.max];
 
-    if (summary.min === summary.max && !isClipped) {
+    if (
+      (summary.min === summary.max && !isClipped) ||
+      singleContinuousValues.has(field)
```

Comment:
If the field is already in the singleContiousValues, we probably don't want to call dispatch again since the key/value are already in the map.  Might want to instead add an else if that just returns the same nulled object to avoid the unnecessary dispatch.